src/libraries/Common/src/System/Security/Cryptography/RsaPaddingProcessor.cs

```diff
@@ -39,7 +39,7 @@ internal sealed class RsaPaddingProcessor
 
         private static readonly byte[] s_digestInfoSha512 =
             {
-                0x30, 0x41, 0x30, 0x0D, 0x06, 0x09, 0x60, 0x86, 0x48,
```

Comment:
This is also used by RSAOpenSsl, too, correct? Is there missing test coverage for this?

Reply:
This is only used by RSAAndroid. RSAOpenSsl uses the OpenSSL APIs to do the PKCS padding.

If we had some tests that had tried to verify a known-correct signed blob with a SHA512-RSA-PKCS signature, then we would have caught this earlier (that's effectively what the test this fixes is doing).

Reply:
Yeah, looks like these were only added for RSAAndroid.

since the second byte should be 0x11 more than the last byte (using the pattern from SHA-2 256/384) this was just a missed update from 384 to 512.

Reply:
Probably means we're missing a static verify case of RSA-SSA-PKCS#1_v1.5 with SHA-512 in the RSA implementation tests.